Sources/GRPC/CallHandlers/BidirectionalStreamingCallHandler.swift

```diff
@@ -92,7 +92,7 @@ public class BidirectionalStreamingCallHandler<
     }
   }
 
-  internal override func sendErrorStatus(_ status: GRPCStatus) {
-    self.callContext?.statusPromise.fail(status)
+  internal override func sendErrorStatusAndMetadata(_ statusAndMetadata: GRPCStatusAndMetadata) {
+    self.callContext?.statusPromise.fail(statusAndMetadata.status)
```

Comment:
We'll also need a test for this to validate that library errors, when transformed, return the extra metadata to the caller.

`testImplementedMethodReconfiguresPipeline` is a good starting point since it sets up an `EmbeddedChannel` and then reconfigures the channel to setup a call handler; you can then just fire an error into the channel and manually read out the response parts.